src/analysis.py

```diff
@@ -116,7 +123,7 @@ def read_input_file(input_file_path):
         sys.exit()
 
     # TODO possibly move this to a config or test script to remove magic values
-    combined_columns = ['name', 'id', 'align', 'eye', 'hair', 'sex', 'gsm','appearances', 'first_appearance', 'year', 'publisher']
+    combined_columns = ['name', 'id', 'align', 'eye', 'hair', 'sex', 'gsm','appearances', 'first_appearance', 'year', 'publisher', 'first_name', 'last_name', 'is_common', 'name_len', 'appear_per_yr']
```

Comment:
I just realized there is another column called "has_last_name".

Reply:
Added that!